src/DependencyInjection/CmfSeoExtension.php

```diff
@@ -34,11 +34,6 @@ class CmfSeoExtension extends Extension
      */
     private $defaultAlternateLocaleProviderId;
 
-    /**
-     * @var bool Whether the content listener is loaded
-     */
-    private $contentListenerEnabled = false;
```

Comment:
was this just unused?

Reply:
yes, and sensio quality guard mentioned it.